content/en/docs/external-add-ons/feast/getting-started.md

```diff
@@ -0,0 +1,67 @@
++++
+title = "Getting Started"
+description = "Getting started with Feast and Kubeflow"
+weight = 20
++++
```

Comment:
We need FEAST folks to review this change.
/assign @woop @sudohainguyen @adchia